src/main/kotlin/com/sourcegraph/cody/edit/sessions/FixupSession.kt

```diff
@@ -181,7 +184,11 @@ abstract class FixupSession(
       // Then they transition to finished.
       CodyTaskState.Finished -> dispose()
       CodyTaskState.Error -> dispose()
-      CodyTaskState.Pending -> {}
+      CodyTaskState.Pending -> {
+        // TODO: This is a hack to allow the unit test command to be recognized.
+        // Currently we only set the task state to Pending for the unit test command.
+        isUnitTestCommand = true
```

Comment:
This looks very hackish 😕 I would rather override the value in the impl (`com.sourcegraph.cody.edit.sessions.TestCodeSession`)

Reply:
thanks for the suggestion! I'm not familiar with kotlin so could use your feedback on the updated approach. Does the updated code looks like what you have in mind?